docs/platform-engineers/addon/intro.md

```diff
@@ -2,16 +2,20 @@
 title: Build Your Own Addon
 ---
 
+A KubeVela addon is a collection that can contain the following two types of files；
+* `Extension configuration file` that defines KubeVela extensibility points, such as [X-definitions](../../getting-started/definition), [UI-Schema](../../reference/ui-schema) or [topology-rules](../../reference/topology-rule).
+* `Resource description file` that defines Kubernetes resource objects. They may be the Kubernetes operator behind a X-definition, or the webserver, etc.
+
 The picture below shows what KubeVela does when an addon is enabled. There are mainly three process:
-* [Addon Registry](./addon-registry) store addons which can be used to share and distribute addons anywhere, it can be any git repo, OSS bucket or OCI registry.
+* [Addon Registry](./addon-registry) store addons which can be used to share and distribute addons anywhere, it can be any git repo, helm repo.
```

Comment:
```suggestion
* [Addon Registry](./addon-registry) store addons which can be used to share and distribute addons anywhere, it can be any git or helm chart repository.
```